Task3_Position_the_Cubes/index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        *{
+            box-sizing: border-box;
+            margin: 0px;
+            padding: 0px;
+        }
+        .c1{
+         border: 2px solid white;
+         background-color: red;
+         height: 100px;
+         width: 100px;
+         position: relative;
+         top: 0px;
+         left: 0px;
+        } 
+        .c2{
+         border: 2px solid white;
+         background-color: green;
+         height: 100px;
+         width: 100px;
+         /* position: relative; */
+         top: 0px;
+         left: 0px;
+         position: absolute;
+         top: 100px;
+         left: 100px;
+         /* bottom: 0px; */

```

Comment:
The bottom property should be removed from this part of the code since it is commented out.